_posts/2016-05-05-cargo-pillars.md

```diff
@@ -0,0 +1,489 @@
+---
+layout: post
+title: "Cargo: predictable dependency management"
+author: Yehuda Katz
+description: "Cargo makes dependency management in Rust easy and predictable"
+---
+
+Cargo's goal is to make modern application package management a core value of
+the Rust programming language.
+
+That concern is driven by a desire to answer questions like:
+
+1. If I want to use `libgit2` in my project, how easy is it?
```

Comment:
"How easy is it to use a library in my project?"

Mentioning `libgit2` is a concrete example, but we don't build on the concrete example later in the article